test/test-debuglet.js

```diff
@@ -120,6 +122,61 @@ describe('Debuglet', function() {
       debuglet.start();
     });
 
+    it('should use GCLOUD_PROJECT in lieu of config.projectId', function(done) {
+      process.env.GCLOUD_PROJECT = '11020304f2934-b';
```

Comment:
+1. But instead of saving restoring, simply add a `projectId` property to the config object below.